api/types/authentication.go

```diff
@@ -460,7 +603,48 @@ const LocalAuthSecretsSchema = `{
 				"pubkey": {"type": "string"}
 			}
 		},
-		"u2f_counter": {"type": "number"}
+		"u2f_counter": {"type": "number"},
+		"mfa": {
+			"type": "array",
+			"items": {
+				"type": "object",
+				"additionalProperties": false,
+				"properties": {
+					"kind": {"type": "string"},
+					"subKind": {"type": "string"},
+					"version": {"type": "string"},
+					"metadata": {
+						"type": "object",
```

Comment:
nit: this should just reference the metadata spec if the JSON schema implementation is capable of references to avoid duplication.